_includes/landing/law-reg-tiles.html

```diff
@@ -0,0 +1,30 @@
+<div class="grid-row grid-gap law-reg-tiles crt-landing--service_animals" markdown="0">
+    <div class="grid-col lp-tile crt-landing--section__item">
+        <div class="crt-landing--reporting_column">
+            <div class="h4 crt-landing--icon_grey"><img src="{{'/assets/images/landing/laws_regs_standards_grey_bg.png' | relative_url}}" alt=""/></div>
+            <div>
+                <h3>
+                    <a href="{{'/law-and-regs/regulations/' | relative_url}}" class="law-reg-tile-link" data-ga-event-name="Learn more regulations">Regulations</a>
+                </h3>
+                <div style="margin-top: 0px" class="no-bullets padding-left-2">
+                <p>DOJ is responsible for issuing regulations under the Americans with Disabilities Act (ADA) that explain the rights of people with disabilities and the obligations of those covered by the laws.</p>
+                </div>
+                <a href="{{'/law-and-regs/regulations/' | relative_url}}" class="usa-button law-reg-tile-link" data-ga-event-name="Learn more regulations">Learn more</a>
+            </div>
+        </div>
+    </div>
```

Comment:
These don't appear to be scaling well to mobile:

<img width="393" alt="image" src="https://github.com/usdoj-crt/beta-ada/assets/15126660/22ee0571-01c8-424b-9458-50ab861aa390">